.github/workflows/update_bundle_snaps.yaml

```diff
@@ -0,0 +1,125 @@
+        required: true
+
+jobs:
+  update-bundle-snaps:
+    name: Update bundle snaps
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Get workflow version
+        id: workflow-version
+        uses: canonical/get-workflow-version-action@v1
+        with:
+          repository-name: canonical/data-platform-workflows
+          file-name: update_bundle_snaps.yaml
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install CLI
+        run: pipx install git+https://github.com/canonical/data-platform-workflows@'${{ steps.workflow-version.outputs.sha }}'#subdirectory=python/cli
+      - name: Free up disk space
+        timeout-minutes: 1
+        run: |
+          printf '\nDisk usage before cleanup\n'
+          df --human-readable
+          # Based on https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
+          rm -r /usr/share/dotnet
+          rm -r /opt/hostedtoolcache/
+          printf '\nDisk usage after cleanup\n'
+          df --human-readable
+      - name: Set up environment
+        timeout-minutes: 60
+        run: |
+          # `--classic` applies to juju 2 snap; ignored for juju 3 snap
+          sudo snap install juju --classic --channel='3.4/stable'
+          sudo snap refresh lxd --channel='latest/stable'
+          sudo adduser "$USER" 'lxd'
+          # `newgrp` does not work in GitHub Actions; use `sg` instead
+          sg 'lxd' -c "lxd waitready"
+          sg 'lxd' -c "lxd init --auto"
+          sg 'lxd' -c "lxc network set lxdbr0 ipv6.address none"
+          sudo iptables -F FORWARD
+          sudo iptables -P FORWARD ACCEPT
+          # shellcheck disable=SC2193
+          # (shellcheck sees it as constant, but GitHub Actions expression is not constant between workflow runs)
+          sg 'lxd' -c "lxc image copy ubuntu:82b997ec581b local: --alias 'juju/ubuntu@22.04/amd64'"
+          mkdir -p ~/.local/share/juju  # Workaround for juju 3 strict snap
+          sg 'lxd' -c "juju bootstrap 'lxd' --config model-logs-size=10G '--agent-version=3.4.5'"
+          juju model-defaults logging-config='<root>=INFO; unit=DEBUG'
+          juju add-model test
+          juju set-model-constraints arch='amd64'
+          sg 'lxd' -c "lxc image list"
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.token }}
+      - name: Deploy bundle and fetch snaps
+        id: update-file
+        timeout-minutes: 60
+        run: |
+          juju deploy ./"${{ inputs.path-to-bundle-file }}"
+          sleep 600 # time to allow the bundle to settle
+          juju status
+          juju exec --all --output snaps.txt -- snap list
+          cat ./snaps.txt
+          output_table=$(cat ./snaps.txt | awk '! /^[0-9]/ && $1 != "Name" { print $1, $3, $4 }' | sort | uniq)
+          canonical_livepatch=$(snap info canonical-livepatch | awk '/^channels:/ {found=1; next} found && $1 != "" {print $1, $4; exit}')
```

Comment:
`canonical-livepatch` snap is not installed right-away on bundle deploy, but `ubuntu-advantage` charm may install it during execution, so we should have it on the list.